tests/unit/CardsSectionUtilsTest.ts

```diff
@@ -35,3 +60,158 @@ describe('getNextBillingDate', () => {
         expect(CardSectionUtils.getNextBillingDate()).toEqual(expectedNextBillingDate);
     });
 });
+
+describe('CardSectionUtils', () => {
+    afterEach(() => {
+        jest.clearAllMocks();
```

Comment:
Can we use `jest.restoreAllMocks` instead of `jest.clearAllMocks`? The reason is because `restoreAllMocks` is the most powerful of the three functions used to cleanup unit test mocks. `clearAllMocks` doesn't restore mock implementations between unit tests, which could lead to the unit tests changing the behavior of subsequent tests due based on the mock implementations of previous tests. `restoreAllMocks` makes each individual test much more sandboxed, which is good. See [SO](https://stackoverflow.com/questions/58151010/difference-between-resetallmocks-resetmodules-resetmoduleregistry-restoreallm) for differences.